maps-compose-utils/src/main/java/com/google/maps/android/compose/kml/manager/MarkerManager.kt

```diff
@@ -208,79 +162,69 @@ public class MarkerManager(
         val markerState = rememberMarkerState(position = position)
         val currentMarkerData = markerData.value
 
-        Marker(
-            state = markerState,
-            alpha = currentMarkerData.alpha,
-            anchor = Offset(currentMarkerData.anchor.x, currentMarkerData.anchor.y),
-            rotation = currentMarkerData.rotation.toFloat(),
-            snippet = currentMarkerData.description,
-            title = currentMarkerData.name,
-            visible = currentMarkerData.visibility,
-            zIndex = currentMarkerData.drawOrder,
-            icon = getIcon(currentMarkerData.icon),
-            onClick = {
-                listener?.onEvent(KmlEvent.Marker.Clicked(markerData.value))
-                true
-            }
-        )
+        if (currentMarkerData.visibility) {
+            Marker(
+                state = markerState,
+                alpha = currentMarkerData.alpha,
+                anchor = Offset(currentMarkerData.anchor.x, currentMarkerData.anchor.y),
+                rotation = currentMarkerData.rotation.toFloat(),
+                snippet = currentMarkerData.description,
+                title = currentMarkerData.name,
+                visible = currentMarkerData.visibility,
+                zIndex = currentMarkerData.drawOrder,
+                icon = getIcon(currentMarkerData.icon),
+                onClick = {
+                    listener?.onEvent(KmlEvent.Marker.Clicked(markerData.value))
+                    true
```

Comment:
was er een reden dat je hier true moet teruggeven

Reply:
het is een lambda die een boolean terug verwacht
`onClick: (Marker) -> Boolean = { false },`
Alleen wordt er niet gespecificeerd wat het verschil tussen true en false is, en merk hier ook geen verschil in